src/index.ts

```diff
@@ -80,6 +94,7 @@ export interface MongoCrypt {
     version: bigint;
     versionStr: string;
   } | null;
+  readonly cryptoHooksProvider: 'js' | 'native_openssl' | null;
```

Comment:
Yeah, it should fall back to these hooks that always throw: https://github.com/mongodb/libmongocrypt/blob/3fd7377d611bd361127f798eca65d9f98f18e228/src/crypto/none.c

Reply:
What I was really wondering was whether we could get rid of the `| null`, but I guess since the dummy crypto functions throw when invoked, the error thrown would be during encryption or decryption, not during construction of a `MongoCrypt`.  Thanks